src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -132,6 +132,15 @@ export default class AirportModel {
         */
         this.airac = null;
 
+        /**
+         * Transition altitude
+         *
+         * @property transitionAltitude
+         * @type {number}
+         * @default null
+        */
+        this.transitionAltitude = null;
```

Comment:
would it make sense to assign this, initially, as 18000? Or are you depending on this to be `null` later on?